apps/web/src/services/dashboards/widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue

```diff
@@ -0,0 +1,315 @@
+                                </span>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </widget-frame>
+</template>
+<script setup lang="ts">
+import type { ComputedRef } from 'vue';
+import {
+    computed, defineExpose, defineProps, nextTick, reactive, ref, toRef, toRefs,
+} from 'vue';
+
+import {
+    PI, PDivider, PDataLoader,
+} from '@spaceone/design-system';
+import dayjs from 'dayjs';
+import { cloneDeep, random } from 'lodash';
+
+import { getRGBFromHex } from '@cloudforet/core-lib';
+
+import { useAmcharts5 } from '@/common/composables/amcharts5';
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+import { red, green } from '@/styles/colors';
+
+import type { DateRange } from '@/services/dashboards/config';
+import WidgetFrame from '@/services/dashboards/widgets/_components/WidgetFrame.vue';
+import type { WidgetProps, WidgetExpose } from '@/services/dashboards/widgets/_configs/config';
+import { SEVERITY_STATUS_MAP } from '@/services/dashboards/widgets/_configs/view-config';
+import { getDateAxisSettings } from '@/services/dashboards/widgets/_helpers/widget-chart-helper';
+import { useWidgetColorSet } from '@/services/dashboards/widgets/_hooks/use-widget-color-set';
+import { useWidgetFrameProps } from '@/services/dashboards/widgets/_hooks/use-widget-frame-props';
+// eslint-disable-next-line import/no-cycle
+import { useWidgetLifecycle } from '@/services/dashboards/widgets/_hooks/use-widget-lifecycle';
+// eslint-disable-next-line import/no-cycle
+import { useWidgetState } from '@/services/dashboards/widgets/_hooks/use-widget-state';
+import type { XYChartData } from '@/services/dashboards/widgets/type';
+
+
+const DATE_FORMAT = 'YYYY-MM';
+const DATE_FIELD_NAME = 'date';
+
+const props = defineProps<WidgetProps>();
+const chartContext = ref<HTMLElement|null>(null);
+const chartHelper = useAmcharts5(chartContext);
+const { colorSet } = useWidgetColorSet({
+    theme: toRef(props, 'theme'),
+    dataSize: computed(() => 1),
+});
+const state = reactive({
+    ...toRefs(useWidgetState(props)),
+    chartData: computed<XYChartData[]>(() => state.data),
+    dateRange: computed<DateRange>(() => {
+        const end = dayjs.utc(state.settings?.date_range?.end).format(DATE_FORMAT);
+        const start = dayjs.utc(end).subtract(11, 'month').format(DATE_FORMAT);
+        return { start, end };
+    }),
+    severityData: computed(() => {
+        const results: any[] = []; // TODO: fix type after get real data
+        Object.values(SEVERITY_STATUS_MAP).forEach((severity) => {
+            if (severity.name === 'pass') return;
+            const currValue = random(0, 200);
+            const prevValue = random(0, 200);
+            results.push({
+                ...severity,
+                value: currValue,
+                diff: currValue - prevValue,
+                rgb: getRGBFromHex(severity.color),
```

Comment:
description: In rgba(), I couldn't use hex colors, so I created this helper to use them.